interface/modules/zend_modules/module/PrescriptionTemplates/src/PrescriptionTemplates/Controller/HtmlTemplatesController.php

```diff
@@ -34,6 +35,10 @@
  */
 class HtmlTemplatesController extends PrescriptionTemplatesController
 {
+    public function __construct(ContainerInterface $container)
```

Comment:
Why do you need the container interface here when you don't do anything with it?